### DISCUSSION TITLE
Confusion about `create()` function

### DISCUSSION THREAD

**Discussion by gregnordin** [pyscript/ltk#15] [General]
@laffra  I'm very interested in ltk but as I try it I believe I am missing something fundamental. Specifically, in the examples, each file has a `create()` function. I don't understand how `create()` is used for a given example. 

For example, when I look through the code that generates https://pyscript.github.io/ltk/, I see instances of `.create()` in the `examples.__init__.py` file in the `items` list. When `examples` is imported, I understand that the code in `__init__.py` will be executed and therefore each `.create()` function will be called, returning whatever `ltk` things it creates. But I don't understand how these are populated in the relevant tabs.

Alternatively, if I take the code from a specific example file and try to use it in my own simple `index.html` file (with pyscript, ltk, etc. properly loaded in the file), I don't know where I should call the `create()` function, and things I have tried have resulted in errors. Any clarification you can provide would be much appreciated. Thanks!

**Comment by laffra** [comment]
If you copy the contents of an example to a new file and then add the
following, it should work:

create().appendTo(ltk.find("body"))

**Reply by gregnordin** [reply]
Ahh, that's what I was missing! Thank you so much! Things work now, and I can see how to add whatever the `create()` function returns to wherever I want on the page by finding the correct location. Makes sense now....

**Comment by laffra** [comment]
👍
